example.rules

```diff
@@ -0,0 +1,9 @@
+# The count of CPUs per node, useful for getting CPU time as a percent of total.
+instance:node_cpu:count = count(node_cpu{mode="idle"}) without (cpu,mode)
+
+# CPU in use by CPU.
+instance_cpu:node_cpu_not_idle:rate2m = sum rate(node_cpu{mode!="idle"}[2m])) without (mode)
+
+# CPU in use by mode.
+instance_mode:node_cpu_not_idle:rate2m = sum rate(node_cpu{mode!="idle"}[2m])) without (cpu)
```

Comment:
Or to put it another way, if this rule is correct then the way we're exposing the metric is incorrect.